examples/python/dynamodb_logstore.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import threading
+
+from pyspark import SparkContext
+from pyspark.sql import Column, DataFrame, SparkSession, SQLContext, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+from multiprocessing.pool import ThreadPool
+import time
+
+"""
+create required dynamodb table with:
+
+$ aws --region us-west-2 dynamodb create-table \
+    --table-name delta_log_test \
+    --attribute-definitions AttributeName=parentPath,AttributeType=S \
+                            AttributeName=filename,AttributeType=S \
+    --key-schema AttributeName=parentPath,KeyType=HASH \
+                AttributeName=filename,KeyType=RANGE \
+    --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5
+
+run this script in root dir of repository
+"""
+
```

Comment:
Can you add an example invocation? e.g. something like what you posted on the doc you sent